source/develop/release-management/features/metrics/metrics-store-installation.html.md

```diff
@@ -66,6 +79,8 @@ to the metrics store machine.
 
 Please follow the installation instructions: [Metrics Store setup on top of OpenShift](https://github.com/ViaQ/Main/blob/master/README-install.md)
 
+**Note:** When running ansible to configure OpenShift, use the ansible-inventy file based on your Openshift version and flavor.
```

Comment:
`ansible-inventory`
Also, use `OpenShift` instead of `Openshift`

Reply:
Done. Thanks.